html/videoroomtest.html

```diff
@@ -71,7 +71,7 @@
 					<button class="btn btn-default" autocomplete="off" id="start">Start</button>
 				</h1>
 			</div>
-			<div class="container" id="details">
```

Comment:
Nitpicking here, but why removing the container class from all those elements in the demo? It's bootstrap stuff and I think we need it for layout reasons.